### PR TITLE
executor: refine log information about slow queries

### PR DESCRIPTION
**Pull request by zz-jason**
1. make the position of log informations stable by not using map
2. log table and index IDs

### CODE REVIEW

**Comment by zz-jason** [comment]
@XuHuaiyu @winoros @coocood PTAL

**Comment by coocood** [comment]
LGTM

File: executor/adapter.go [hunk 1]
```diff
@@ -339,18 +340,17 @@ func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
...
+
 	if costTime < threshold {
-		logEntry.WithField("type", "query").WithField("succ", succ).Debugf("query")
+		logutil.SlowQueryLogger.Debugf(
+			"[QUERY] cost_time:%v succ:%v connection_id:%v txn_start_ts:%v database:%v table_ids:%v index_ids:%v sql:%v",
+			costTime, succ, connID, txnTS, currentDB, tableIDs, indexIDs, sql)
 	} else {
-		logEntry.WithField("type", "slow-query").WithField("succ", succ).Warnf("slow-query")
+		logutil.SlowQueryLogger.Warnf(
+			"[SLOW_QUERY] cost_time:%v succ:%v connection_id:%v txn_start_ts:%v database:%v table_ids:%v index_ids:%v sql:%v",
```
Comment (shenli) [inline executor/adapter.go]: Do we need to keep the tag unchanged? Some users are used to grep 'slow-query'.

**Comment by XuHuaiyu** [comment]
LGTM

File: executor/adapter.go (hunk 1 above)
Comment (zz-jason) [reply executor/adapter.go]: I think both tags are OK, I just change it to be the same form of other logs such as "EXPENSIVE_QUERY", "TIME_COP_TASK"

File: executor/adapter.go [hunk 2]
```diff
@@ -339,18 +340,17 @@ func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
...
+	indexIDs := strings.Replace(fmt.Sprintf("%v", a.Ctx.GetSessionVars().StmtCtx.IndexIDs), " ", ",", -1)
+
 	if costTime < threshold {
-		logEntry.WithField("type", "query").WithField("succ", succ).Debugf("query")
+		logutil.SlowQueryLogger.Debugf(
+			"[QUERY] cost_time:%v succ:%v connection_id:%v txn_start_ts:%v database:%v table_ids:%v index_ids:%v sql:%v",
+			costTime, succ, connID, txnTS, currentDB, tableIDs, indexIDs, sql)
 	} else {
-		logEntry.WithField("type", "slow-query").WithField("succ", succ).Warnf("slow-query")
+		logutil.SlowQueryLogger.Warnf(
```
Comment (tiancaiamao) [inline executor/adapter.go]: This would make the slow query log file messed up.

**Pull request merged**